metascoop/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.23.1
 require gopkg.in/yaml.v3 v3.0.1
 
 require (
-	github.com/google/go-github/v66 v66.0.0
+	github.com/google/go-github/v64 v64.0.0
 	github.com/google/go-github/v66 v66.0.0
```

Comment:
Renovate has been adding new dependency versions instead of updating existing lines, this should be removed for now and we have to figure out how to improve Renovate's handling.